src/main/webapp/WEB-INF/tags/shared/commentsPanel.tag

```diff
@@ -21,65 +21,89 @@
 </c:choose>
 <div class="panel panel-primary">
     <div class="panel-heading">
+        <div class='display-icon pull-right'>
+            <span class="glyphicon ${'glyphicon-chevron-up'} pull-right"></span>
+        </div>
         <strong>${bodyTitle}</strong>
     </div>
-    <div class="panel-body">
+    <div class="panel-collapse collapse in">
```

Comment:
@JosephineKwa I edited collapse function for this, to be able to work on both .panel-body and .panel-collapse .  But .panel-body is not working as smooth as .panel-collapse. Should I go for .panel-body ?